src/core/server/metrics/metrics_service.ts

```diff
@@ -62,6 +83,15 @@ export class MetricsService
     return this.service;
   }
 
+  private reportOpsMetrics(analytics: AnalyticsServiceSetup) {
+    this.service
+      ?.getOpsMetrics$()
+      // @ts-expect-error-error
+      // Argument of type 'OpsMetrics' is not assignable to parameter of type 'Record<string, unknown>'.
+      // Index signature for type 'string' is missing in type 'OpsMetrics'.ts(2345)
+      .subscribe((metrics: OpsMetrics) => analytics.reportEvent('core-ops_metrics', metrics));
+  }
```

Comment:
Unfortunately, it is a known bug in TS https://github.com/microsoft/TypeScript/issues/15300

We can get around it by spreading the metrics
```suggestion
  private reportOpsMetrics(analytics: AnalyticsServiceSetup) {
    this.service
      ?.getOpsMetrics$()
      .subscribe((metrics: OpsMetrics) => analytics.reportEvent('core-ops_metrics', { ...metrics }));
  }
```

Reply:
Ah! Great!